.github/pull_request_template.md

```diff
@@ -15,16 +15,12 @@ This should be a short TL;DR that includes the purpose of the PR.
 ### :link: External links
 
 <!-- Issues, RFC, etc. -->
+Jira ticket: [if it applies]
```

Comment:
```suggestion
Jira ticket: [HDS-XXX](https://hashicorp.atlassian.net/browse/HDS-XXX)
```
Suggestion that this _might_ be easier but no strong opinion (reminds me this is supposed to work automagically but doesn't)

Reply:
I like this suggestion